evmlab/opcodes.py

```diff
@@ -127,4 +130,37 @@
 BALANCE_SUPPLEMENTAL_GAS = 380
 CALL_CHILD_LIMIT_NUM = 63
 CALL_CHILD_LIMIT_DENOM = 64
-SUICIDE_SUPPLEMENTAL_GAS = 5000
\ No newline at end of file
+SUICIDE_SUPPLEMENTAL_GAS = 5000
+
+
+def parseCode(code):
+    code = remove_0x_head(code)
+    codes = [c for c in decode_hex(code)]
+
+    # Remove the last 43 bytes, which is the contract metadata
+    # TODO: Make this conditional, as it's assumed the metadata exists.
+    # codes = codes[:-43]
```

Comment:
I'm not sure about this - I want it to be as close to the metal as possible. What's the reason to remove 43 bytes of data? I mean, does it break anything to have it there?

Reply:
ah, I forgot to remove it. it's is commented out currently. It was in the truffle debugger code that I ported over. However, I couldn't find a reason to keep it, so I commented it out.

I'll remove it when I rebase